README.md

```diff
@@ -0,0 +1,38 @@
+## Overview
+This repository comprises prebuilt healthcare-related services that are ready for immediate use. Users have the flexibility to deploy and run these services in their respective environments. Optionally, these services can be deployed in [Choreo](https://console.choreo.dev/home) as well. For further details, refer to the documentation of the individual services.
+
+## Available Services
+### Conformance
+#### [Metadata Service](conformance/metadata-fhirr4-service/)
+This service facilitates the exposure of a capability statement describing the current operational functionality of a FHIR server.
+
+#### [SMART Configuration Service](conformance/smart-config-fhirr4-service/)
+This service can be used to expose the discovery document containing authorization endpoint URLs and SMART-features supported by a FHIR server.
+
+### Transformation
+#### [HL7v2 to FHIR](transformation/v2-to-fhirr4-service/)
+Transform HL7v2 messages into FHIR resources using this service.
+
+#### [CCDA to FHIR](transformation/ccda-to-fhirr4-service/)
+This service can be usde to transform C-CDA messages into FHIR resources.
+
+### EHR Related
+#### EPIC Services
+These services can be used to integrate with an Epic instance and expose Epic FHIR APIs.
+
+- [Epic FHIR R4 Administration Service](ehr-connectivity/epic-fhirr4-administration-api-service/)
+- [Epic FHIR R4 Clinical Service](ehr-connectivity/epic-fhirr4-clinical-api-service/)
+- [Epic FHIR R4 Diagnostics Service](ehr-connectivity/epic-fhirr4-diagnostics-api-service/)
+- [Epic FHIR R4 Financial Service](ehr-connectivity/epic-fhirr4-financial-api-service/)
+- [Epic FHIR R4 Medications Service](ehr-connectivity/epic-fhirr4-medications-api-service/)
+- [Epic FHIR R4 Workflow Service](ehr-connectivity/epic-fhirr4-workflow-api-service/)
+
+### Miscellaneous
+#### [Authorization Service](miscellaneous/authz-fhirr4-service/)
+Apply basic patient and privileged user-based authorization policies for FHIR APIs using this service.
```

Comment:
This service basically evaluate the policies?

Reply:
Yes. And we can use this evaluation to secure a FHIR API. So ultimately the policies get applied to the API.